shared/login/signup/common.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import * as React from 'react'
+import * as Constants from '../../constants/signup'
+import {
+  Box2,
+  HeaderHocHeader,
+  Avatar,
+  WaitingButton,
+  Input as CommonInput,
+  ButtonBar,
+} from '../../common-adapters'
+import {styleSheetCreate, isMobile, globalMargins} from '../../styles'
+
+type Props = {
+  children: React.Node,
+  onBack: () => void,
+}
+
+export const Wrapper = (props: Props) => (
+  <Box2 direction="vertical" fullWidth={true} fullHeight={true}>
+    <Box2
+      direction="vertical"
+      fullWidth={true}
+      fullHeight={true}
+      centerChildren={true}
+      style={styles.wrapper}
+      gap={isMobile ? 'xtiny' : 'small'}
+    >
+      {props.children}
+    </Box2>
+    <HeaderHocHeader onBack={props.onBack} headerStyle={styles.header} />
```

Comment:
this works but the bottom border still ends to the left:
<img width="387" alt="image" src="https://user-images.githubusercontent.com/11968340/41997115-0d43f616-7a25-11e8-9687-db27466d5170.png">

Reply:
k, i  think we should just remove the border anyways, we're not using a title